README.md

```diff
@@ -68,6 +68,6 @@ source before proceeding with the install.
 [slack]: http://slack.opengee.org
 [slack-img]: https://img.shields.io/badge/chat-on%20slack-ff69b4.svg
 [travis]: https://travis-ci.org/google/earthenterprise/builds
-[travis-img]: https://travis-ci.org/google/earthenterprise.svg?branch=master
+[travis-img]: https://travis-ci.org/google/earthenterprise.svg?branch=release_5.3.3
```

Comment:
I think I've asked about this before, but do we really want this pointed at the last release? It seems like we should be indicating whether the master build is passing.